dbms/src/Storages/Transaction/tests/gtest_new_kvstore.cpp

```diff
@@ -154,6 +154,43 @@ TEST_F(RegionKVStoreTest, KVStoreFailRecovery)
     }
 }
 
+TEST_F(RegionKVStoreTest, KVStoreInvalidWrites)
+{
+    auto ctx = TiFlashTestEnv::getGlobalContext();
+    {
+        auto region_id = 1;
+        {
+            initStorages();
+            KVStore & kvs = getKVS();
+            proxy_instance->bootstrap_table(ctx, kvs, ctx.getTMTContext());
+            proxy_instance->bootstrap(kvs, ctx.getTMTContext(), region_id);
+        }
+        {
+            KVStore & kvs = getKVS();
+            proxy_instance->bootstrap(kvs, ctx.getTMTContext(), region_id);
+            MockRaftStoreProxy::FailCond cond;
+
+            auto kvr1 = kvs.getRegion(region_id);
+            auto r1 = proxy_instance->getRegion(region_id);
+            ASSERT_NE(r1, nullptr);
+            ASSERT_NE(kvr1, nullptr);
+            ASSERT_EQ(r1->getLatestAppliedIndex(), kvr1->appliedIndex());
+            {
+                r1->getLatestAppliedIndex();
+                std::string k = "7480000000000001FFBD5F720000000000FAF9ECEFDC3207FFFC";
```

Comment:
can you add some comment about the meaning of the `k` for easier later maintenance.

Reply:
This is just an invalid key(maybe truncated) TiKV sent...